src/Misc/MiddleWare.cpp

```diff
@@ -586,9 +613,29 @@ class mw_dispatcher_t : public master_dispatcher_t
 
 class MiddleWareImpl
 {
+public:
+    struct waveTablesToGenerateStruct
```

Comment:
Yeah, this makes more sense after reading the timing diagram. So this is just the queued operation.

Reply:
Yes, that struct is a queue element.